examples/universal-cnf/README.md

```diff
@@ -1,7 +1,126 @@
 # The Universal CNF
 
-Universal CNF is an attempt to implement a swiss-army knife container, which implements basic networking functions useful for demoing NSM. It is currently heavily based in VPP and vppagent.
+Universal CNF (UCNF) is an attempt to implement a swiss-army knife container, which implements basic networking functions useful for demoing NSM. It is currently heavily based on [VPP](https://github.com/FDio/vpp) and [vppagent](https://github.com/ligato/vpp-agent).
 
 ## Configuration
 
-TBD
\ No newline at end of file
+UCNF can be deployed and configured to implement various cloud-native netowrking scenarios. For this, it exposes a configuration file interface, where the solution integrator can program it.
```

Comment:
I think "For that" is more correct

Reply:
Great write up! Found one small typo in "networking", but other than that it looks great 👍